microsoft/runbook/debug.yml

```diff
@@ -0,0 +1,21 @@
+# this runbook overwrites test criteria and uses to debug test cases by name pattern.
+include:
+  - path: $(origin)
```

Comment:
Would it make sense for this runbook to be instead setup as an alternative tier?

Reply:
Good point, add one more runbook as "debug" tier. This one is kept, because if someone use the totally different test suites than microsoft, the top-level runbook can work. For example, the "examples" runbook doesn't include the Microsoft test suites, and it can use the top-level debug runbook.